kafka_log/README.md

```diff
@@ -3,7 +3,7 @@ This agent collects data from kafka and sends it to Insightfinder.
 ## Installing the Agent
 
 ### Required Dependencies:
-1. Python 3.x 
+1. Python >= 3.6.8
```

Comment:
For the offline packages, we should hard require python 3.6.8 which is the default python3 for RHEL 7/8